### PR TITLE
⬆️  Update appcues sdk version to 2.1.2

### PR DESCRIPTION
**Pull request by andretortolano**

### CODE REVIEW

File: example/android/build.gradle [hunk 1]
```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```
Comment (andretortolano) [inline example/android/build.gradle:29]: not sure why this changed. maybe flutter version that I ran flutter pub get ?

File: example/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -204,6 +204,7 @@
 			files = (
 			);
 			inputPaths = (
+				"${TARGET_BUILD_DIR}/${INFOPLIST_PATH}",
```
Comment (andretortolano) [inline example/ios/Runner.xcodeproj/project.pbxproj:207]: not sure why this was added. is that ok?

File: example/lib/src/app.dart [hunk 3]
```diff
@@ -94,7 +94,7 @@ class _ExampleState extends State<Example> {
     AppcuesOptions options = AppcuesOptions();
     options.logging = true;
     await Appcues.initialize(
-        'APPCUES_ACCOUNT_ID', 'APPCUES_APPLICATION_ID', options);
+        '103523', '4b6b796f-1313-440a-b356-ee4b7abe931c', options);
```
Comment (andretortolano) [inline example/lib/src/app.dart:97]: will remove

File: example/ios/Podfile.lock [hunk 4]
```diff
@@ -25,11 +25,11 @@ EXTERNAL SOURCES:
...
-  appcues_flutter: 60a9857658b86ade3145d6325345bbfb48c37205
+  Appcues: 6293ef6ca66be2e37f9b02248d87de5cbd569a98
+  appcues_flutter: 569d1096aa949dc97caee5b41ae62470201654e3
   Flutter: f04841e97a9d0b0a8025694d0796dd46242b2854
   uni_links: d97da20c7701486ba192624d99bffaaffcfc298a
 
 PODFILE CHECKSUM: 69a94f836326f2bab375ad9c9e93cae9179c7221
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.12.0
```
Comment (iujames) [inline example/ios/Podfile.lock:35]: this part should not change, let's keep on 1.12.1

File: example/ios/Runner.xcodeproj/project.pbxproj (hunk 2 above)
Comment (iujames) [reply example/ios/Runner.xcodeproj/project.pbxproj:207]: pulled down local and it ran with no problem. I'm not really sure what this is, but seems harmless.

File: example/android/build.gradle (hunk 1 above)
Comment (iujames) [reply example/android/build.gradle:29]: I'm not sure, but this also ran fine locally and did not change on my machine, so seems fine.

**Review by iujames — COMMENTED** [review]
I think good as long as we can keep on same cocoapods version please

**Review by iujames — APPROVED** [review]

**Pull request merged**